frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -141,6 +142,17 @@ export const DiskModal = withHandlePromise((props: DiskModalProps) => {
     combinedDisk.getStorageClassName() || '',
   );
 
+  React.useEffect(() => {
+    if (!isEditing) {
```

Comment:
ok seems reasonable, but let's check if isLoaded here

Reply:
this component supports passing half initialized values when creating so lets's also check if storageClassName is not empty as well

Reply:
hm, this should be combined with storageClassConfigMap useEffect since they depend on each other and should set values accordingly

Reply:
added the checks for isLoaded and storageClassName +1
moved inside storageClassConfigMap useEffect +2 :-)